include/ps/kv_app.h

```diff
@@ -425,6 +431,26 @@ struct KVServerDefaultHandle {
 
 ///////////////////////////////////////////////////////////////////////////////
 
+template <typename Val>
+void KVServer<Val>::RegisterRecvBuffer(int worker_id, SArray<Key>& keys,
+                                       const SArray<Val>& vals,
+                                       const SArray<int>& lens,
+                                       int cmd) {
+  Message msg;
+  msg.meta.request = true;
+  msg.meta.push = true;
+  msg.meta.head = cmd;
+  msg.meta.sender = worker_id;
```

Comment:
will it be a plain worker id, or worker id combined with GPU index?

Reply:
Currently, it's a plain worker id